.travis.yml

```diff
@@ -1,9 +1,10 @@
 # whitelist
 branches:
   only:
-      - u/fjammes/DM-1146 
+      - master
+      - tickets/DM-5319
 
-sudo: required
+sudo: false
```

Comment:
I think `sudo` technically should be `required`.  While it seems to work like this, the job output does show sudo being used.